README.md

```diff
@@ -20,6 +20,7 @@ Available addons
 addon | version | summary
```

Comment:
You don't even need to change this one, because it's automatically updated with an script.